src/instrumenter/instrumentation_context.ts

```diff
@@ -467,6 +471,52 @@ export class InstrumentationContext {
         }
     }
 
+    addStringLiteralToAdjust(lit: Literal, loc: ASTNode): void {
+        this.litAdjustMap.set(lit, loc);
+    }
+
+    adjustStringLiterals(
+        contents: string,
+        units: SourceUnit,
+        fileInd: number,
+        srcMap: SrcRangeMap
+    ): string {
+        for (const lit of units.getChildrenByType(Literal)) {
+            if (!(lit.kind === LiteralKind.String && this.litAdjustMap.has(lit))) {
+                continue;
+            }
+
+            const targetNode = this.litAdjustMap.get(lit) as ASTNode;
```

Comment:
We traversing all literals in source unit here and skipping ones that are not in `this.litAdjustMap`. Seem to me that we can use following shortcut more effectively:
```typescript
for (const [lit, targetNode] of this.litAdjustMap) {
   // ...
}
```

Reply:
P.S.: Also `units` is actually a single source unit.

Reply:
Fixed in [029c9c3](https://github.com/ConsenSys/scribble/pull/211/commits/029c9c3117655fdbc0f0f348bcd11e354e5ce2ad)